### PR TITLE
Update architect-orb to 2.3.0

### PR DESCRIPTION
**Pull request by ubergesundheit**
Towards giantswarm/giantswarm#15898

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -15,7 +15,15 @@ require (
 	github.com/giantswarm/versionbundle v0.2.0
 	github.com/prometheus/client_golang v1.9.0
 	github.com/spf13/viper v1.7.1
-	k8s.io/api v0.17.2
-	k8s.io/apimachinery v0.17.2
-	k8s.io/client-go v0.17.2
+	k8s.io/api v0.18.6
+	k8s.io/apimachinery v0.18.6
+	k8s.io/client-go v0.18.6
```
Comment (ubergesundheit) [inline go.mod:20]: These were automatically updated after running `go mod tidy`

**Comment by ubergesundheit** [comment]
Hello @giantswarm/team-rocket!

architect-orb >= 2.1.0 contains an important fix regarding catalog urls. Can you take a look at https://github.com/giantswarm/flannel-operator/pull/243/commits/763d90f7877e7051d807af3d1febf1c1d0c182b3#r588554699 and https://circleci.com/gh/giantswarm/flannel-operator/1215?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link ?

Thanks!

File: go.mod (hunk 1 above)
Comment (tfussell) [reply go.mod:20]: Can we just revert all of the `go.mod` changes? We weren't planning to update this operator to kubernetes 1.18 client libraries.

Comment (ubergesundheit) [reply go.mod]: Yes, reverted and CI is green :)

**Review by tfussell — APPROVED** [review]

**Pull request merged**